### DISCUSSION TITLE
Concise way to specify font-size + line-height

### DISCUSSION THREAD

**Discussion by ivanchenhz** [tailwindlabs/tailwindcss#5782] [Ideas]
In our project, we like to using different line-height for the same font-size,

So in the code we would have to do stuff like
```html
<div class="text-sm leading-[22px]"></div>
<div class="text-sm leading-normal"></div>
```

Could we add feature like what we have done for the color + opacity?
```html
<div class="text-red-100/50"></div>

<div class="text-sm/22px text-[14px/22px] text-[14px/1.5]"></div>
```

**Comment by y1n0** [comment]
came here to suggest this.. I always unthinkingly try to write this combination of font size / line height.. it feels so natural to the point I thought I saw it in the documentation. This definitely something we should be able to do
